### ISSUE TITLE
Pangolin X11: Failed to open X display

### ISSUE THREAD

**Issue by POTATO-30** [PJLab-ADG/SensorsCalibration#108]
Have you ever encountered this problem？Will you help me？
terminate called after throwing an instance of 'std::runtime_error'
  what():  Pangolin X11: Failed to open X display
Aborted(core dumped)

**Comment by maxpepi** [comment]
Yes, try running `x host +`  in a separate Terminal in order to disable the access control.

**Comment by POTATO-30** [comment]
> Yes, try running `x host +` in a separate Terminal in order to disable the access control.
Thank you very much for your answer. I tried to run this command on a new terminal and it was prompted that ‘access control disabled,clients can connect from any host', but this issue still exists.

**Comment by maxpepi** [comment]
have you checked the points in https://github.com/PJLab-ADG/SensorsCalibration/issues/7 ?

**Comment by POTATO-30** [comment]
> have you checked the points in #7 ?
Thank you very much for your answer. I have carefully read this discussion and learned to build an environment step by step on my own host. After a day of hard work, I have finally succeeded! Thank you for your continuous response.

**Comment by maxpepi** [comment]
Awesome! Glad it worked out.

**Comment by AbdullahJirjees** [comment]
@maxpepi Can you share with me you how did you solved the issue? I tried the xhost + and it still showing the same issue?
